modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,270 @@
+  Controller:
+    Processors:
+    Sharding:
+  Grafana:
+    Enabled:  false
+    Ingress:
+      Enabled:  false
+    Route:
+      Enabled:  false
+  Ha:
+    Enabled:  false
+  Initial SSH Known Hosts:
+  Monitoring:
+    Enabled:  false
+  Notifications:
+    Enabled:  false
+  Prometheus:
+    Enabled:  false
+    Ingress:
+      Enabled:  false
+    Route:
+      Enabled:  false
+  Rbac:
+  Redis:
+  Repo:
+  Server:
+    Autoscale:
+      Enabled:  false
+    Grpc:
+      Ingress:
+        Enabled:  false
+    Ingress:
+      Enabled:  false
+    Route:
+      Enabled:  false
+    Service:
+      Type:  
+  Tls:
+    Ca:
+Status:
+  Application Controller:      Running
+  Application Set Controller:  Unknown
+  Phase:                       Available # <1>
+  Redis:                       Running
+  Repo:                        Running
+  Server:                      Running
+  Sso:                         Unknown
+Events:                        <none>
+----
+<1> The status indicates that the cluster-scoped Argo CD instance is healthy and available.
++
+The {gitops-title} Operator creates the following default cluster roles and manages them:
++
+* `<argocd_name>-<argocd_namespace>-argocd-application-controller` aggregated cluster role
+* `<argocd_name>-<argocd_namespace>-argocd-application-controller-view`
+* `<argocd_name>-<argocd_namespace>-argocd-application-controller-admin` 
+
+. Verify that the Operator has created the default cluster roles and cluster role bindings for the Argo CD Application Controller and Argo CD server components by running the following commands:
++
+[source,terminal]
+----
+$ oc get ClusterRoles -l app.kubernetes.io/part-of=argocd
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                           CREATED AT
+example-spring-petclinic-argocd-application-controller         2024-08-14T08:20:58Z
+example-spring-petclinic-argocd-application-controller-admin   2024-08-14T09:08:38Z
+example-spring-petclinic-argocd-application-controller-view    2024-08-14T09:08:38Z
+example-spring-petclinic-argocd-server                         2024-08-14T08:20:59Z
+----
++
+[source,terminal]
+----
+$ oc get ClusterRoleBindings -l app.kubernetes.io/part-of=argocd
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                     ROLE                                                                 AGE
+example-spring-petclinic-argocd-application-controller   ClusterRole/example-spring-petclinic-argocd-application-controller   54m
+example-spring-petclinic-argocd-server                   ClusterRole/example-spring-petclinic-argocd-server                   54m
+----
++
+The cluster role bindings for the `view` and `admin` cluster roles are not created. This is because the cluster role bindings only add permissions to the aggregated cluster role and do not directly configure permissions for the Argo CD Application Controller.
```

Comment:
LGTM, just a suggestion, we can rephrase this line, feel free to ignore if you disagree,.  

"This is because the cluster role bindings only add permissions to the aggregated cluster role and do not directly configure permissions for the Argo CD Application Controller." =>
"This is because they don’t directly configure permissions to the Argo CD Application Controller, instead they add permissions to aggregated cluster role which  has a cluster role binding."

Reply:
Thanks @jparsai. Addressed as discussed. Thanks!